package.json

```diff
@@ -7,28 +7,31 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint --ext js --ext jsx src",
-    "plug": "node injectors/index.js inject --no-exit-codes",
-    "unplug": "node injectors/index.js uninject --no-exit-codes",
-    "inject": "node injectors/index.js inject --no-exit-codes",
-    "uninject": "node injectors/index.js uninject --no-exit-codes"
+    "plug": "ts-node injectors/index.ts inject --no-exit-codes",
+    "unplug": "ts-node injectors/index.ts uninject --no-exit-codes",
+    "inject": "ts-node injectors/index.ts inject --no-exit-codes",
+    "uninject": "ts-node injectors/index.ts uninject --no-exit-codes"
```

Comment:
I think it's better to not use ts-node and instead compile it. This will make it run faster since it doesn't need to be built every time.

Reply:
true, although the injector is only going to be run once, we have incremental complication enabled (not sure if ts-node takes advantage of that,) and the built-in update system isn't currently setup to rebuild the project IIRC (which is going to be something to do down the line)

Reply:
Well if more of the project is going to be in TypeScript in the future then it will be run every time, so might as well get set up good now.

Reply:
hmmm, I guess, I already was planing to do it in a seperate PR